From: Tetsu Yamamoto <tyamamot@redhat.com>
Date: Mon, 4 Aug 2008 15:48:36 -0400
Subject: [xen] ia64: mark resource list functions __devinit
Message-id: 20080804194836.8452.76657.sendpatchset@pq0-1.lab.bos.redhat.com
O-Subject: [RHEL5.3 PATCH 8/10] xen-ia64: Mark resource list functions __devinit
Bugzilla: 430219
RH-Acked-by: Bill Burns <bburns@redhat.com>

bz430219
# HG changeset patch
# User Alex Williamson <alex.williamson@hp.com>
# Date 1185814984 21600
# Node ID e92816da20760df8b8716bbd14f0aec3d2729264
# Parent  8f0c93df3e113058f0b582b06ce7d3d897ccc9d6
[IA64] Mark resource list functions __devinit

Signed-off-by: Alex Williamson <alex.williamson@hp.com>

diff --git a/arch/ia64/pci/pci.c b/arch/ia64/pci/pci.c
index 8d272a1..e371c7c 100644
--- a/arch/ia64/pci/pci.c
+++ b/arch/ia64/pci/pci.c
@@ -347,7 +347,7 @@ pcibios_setup_root_windows(struct pci_bus *bus, struct pci_controller *ctrl)
 }
 
 #ifdef CONFIG_XEN
-static void
+static void __devinit
 __cleanup_issue_list(struct list_head *top)
 {
 	ioremap_issue_list_t *ptr, *tmp_ptr;
@@ -358,7 +358,7 @@ __cleanup_issue_list(struct list_head *top)
 	}
 }
 
-static int
+static int __devinit
 __add_issue_list(unsigned long start, unsigned long end, struct list_head *top)
 {
 	ioremap_issue_list_t *ptr, *new;
@@ -416,7 +416,7 @@ __add_issue_list(unsigned long start, unsigned long end, struct list_head *top)
 	return 0;
 }
 
-static int
+static int __devinit
 __make_issue_list(struct resource *ptr, struct list_head *top)
 {
 	int ret;
@@ -441,7 +441,7 @@ __make_issue_list(struct resource *ptr, struct list_head *top)
 	return 0;
 }
 
-static void
+static void __devinit
 __compress_issue_list(struct list_head *top)
 {
 	ioremap_issue_list_t *ptr, *tmp_ptr, *next;
@@ -467,7 +467,7 @@ __compress_issue_list(struct list_head *top)
 	}
 }
 
-static int
+static int __devinit
 __issue_ioremap(struct list_head *top)
 {
 	ioremap_issue_list_t *ptr, *tmp_ptr;
@@ -489,7 +489,7 @@ __issue_ioremap(struct list_head *top)
 	return 0;
 }
 
-static int
+static int __devinit
 do_ioremap_on_resource_list(struct resource *top)
 {
 	LIST_HEAD(ioremap_issue_list_top);
